### PR TITLE
Burak-recaptchas

### PR DESCRIPTION
**Pull request by myniqx**
https://github.com/freedomcombination/monorepo/issues/980

### CODE REVIEW

File: apps/api/src/api/feedback/controllers/custom.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+export default {
+  async sendEmail(ctx: Context) {
+    await checkRecaptcha(ctx)
+
+    const email = ctx.request.body
+
+    try {
+      await strapi.plugins['email'].services.email.send(email)
+    } catch (error) {
+      console.error(error)
```
Comment (7alip) [inline apps/api/src/api/feedback/controllers/custom.ts:13]: Can return a proper Strapi error

File: apps/dashboard/public/locales/en/common.json [hunk 2]
```diff
@@ -337,5 +337,6 @@
   "form.uploader.success": "Success",
   "form.uploader.copied": "Copied to clipboard",
   "form.uploader.old": "Get old images you uploaded before",
-  "form.uploader.new": "Upload new images"
+  "form.uploader.new": "Upload new images",
+  "about.content": "Art Station is a group where people who took emigrated in the Netherlands, who are interested in art, meet both online and physically, share their experiences with each other, share information on modern and traditional arts, and organize artistic activities at the same time."
```
Comment (7alip) [inline apps/dashboard/public/locales/en/common.json:341]: Would be nice to have it alphabetically ordered

File: apps/api/src/api/profile/controllers/profile.ts [hunk 3]
```diff
@@ -34,5 +34,10 @@ export default factories.createCoreController('api::profile.profile', () => {
 
       return this.transformResponse(sanitizedResults, { pagination })
     },
+    async create(ctx) {
+      await checkRecaptcha(ctx)
```
Comment (7alip) [inline apps/api/src/api/profile/controllers/profile.ts:38]: In `services/src/api/register.ts` we create profile without recaptcha. This part may fail now.

```ts
await Mutation.post('profiles', body, response.data?.jwt)
```

**Review by 7alip — COMMENTED** [review]
Thanks. Remember running `yarn format`

File: apps/api/src/api/feedback/controllers/custom.ts (hunk 1 above)
Comment (myniqx) [reply apps/api/src/api/feedback/controllers/custom.ts:13]: let strapi throw proper error then.

**Review by 7alip — APPROVED** [review]
Thanks

**Pull request merged**